### PR TITLE
add r-installr

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/r-installr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    command performs the following: finding the latest R version, downloading it, running
+    the installer, deleting the installation file, copy and updating old packages to
+    the new R installation.'
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - FernandezMathieu
```
Comment (dbast) [inline recipes/r-installr/meta.yaml:54]:
```suggestion
    - conda-forge/r
    - FernandezMathieu
```

File: recipes/r-installr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    - r-stringr
+
+test:
+  commands:
+    - $R -e "library('installr')"           # [not win]
+    - "\"%R%\" -e \"library('installr')\""  # [win]
+
+about:
+  home: http://talgalili.github.io/installr/, https://github.com/talgalili/installr/, http://www.r-statistics.com/tag/installr/
+  license: GPL-2
```
Comment (dbast) [inline recipes/r-installr/meta.yaml:40]:
```suggestion
  license: GPL-2.0-only
```

File: recipes/r-installr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/installr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/installr/installr_{{ version }}.tar.gz
+  sha256: beec512d131572cc80998817aea540b99aa86048942ec49fdecbd0658f58bfdb
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [not win]
```
Comment (dbast) [inline recipes/r-installr/meta.yaml:18]:
```suggestion
  noarch: generic
```

**Comment by dbast** [comment]
uh. this is indeed windows only and not noarch.

Just being curious ... what do you use installr for?... with conda at hand it should not be required.

**Comment by FernandezMathieu** [comment]
To be honest, i don't know.  
I'm just uploading a bunch of packages needed by my data scientist team. 

After looking at the package's details, it seems to be obvious it's only for windows. Sorry for the mistake.

**Pull request closed without merging**